eventhandler_test.go

```diff
@@ -87,7 +87,7 @@ func TestHandleConfigureMonitoringTriggeredEvent(t *testing.T) {
 	err = HandleConfigureMonitoringTriggeredEvent(ddKeptn, *incomingEvent, data)
 
 	if err != nil {
-		t.Errorf("Error: " + err.Error())
+		t.Errorf("Error: %s", err.Error())
```

Comment:
You can merge `t.Errorf` and `t.Fail` with `t.Fatalf`, but at this point I would suggest to use a `t.Fatal(err)` as it is very basic to make the test fail and report the error

Reply:
Ok. Thanks for the remarks.